src/EditorFeatures/Core/Implementation/InlineRename/Dashboard/Dashboard.xaml

```diff
@@ -31,6 +31,10 @@
                 <Setter Property="Foreground" Value="{DynamicResource {x:Static utilities:CodeAnalysisColors.CheckBoxTextBrushKey}}"/>
             </Style>
 
+            <Style TargetType="TextBlock">
+                <Setter Property="Foreground" Value="{StaticResource ForegroundText}"/>
```

Comment:
`StaticResource` is sufficient?  It doesn't need `DynamicResource` to reflect changes?